trainee_template/marginal.amlg

```diff
@@ -607,39 +646,39 @@
 										min
 											(retrieve_from_entity
 												(first (contained_entities
-													(list (query_min feature 1))
+													(append filtering_queries (query_min feature 1))
 												))
 												feature
 											)
 										max
 											(retrieve_from_entity
 												(first (contained_entities
-													(list (query_max feature 1))
+													(append filtering_queries (query_max feature 1))
 												))
 												feature
 											)
 										mean
-											(compute_on_contained_entities (list
+											(compute_on_contained_entities (append filtering_queries
 												(query_not_equals feature (null))
 												;calculate arithmetic average
 												(query_generalized_mean feature 1 weight_feature)
 											))
 										median
-											(compute_on_contained_entities (list (query_quantile feature 0.5 weight_feature)))
+											(compute_on_contained_entities (append filtering_queries (query_quantile feature 0.5 weight_feature)))
 										percentile_25
-											(compute_on_contained_entities (list (query_quantile feature 0.25 weight_feature)))
+											(compute_on_contained_entities (append filtering_queries (query_quantile feature 0.25 weight_feature)))
 										percentile_75
-											(compute_on_contained_entities (list (query_quantile feature 0.75 weight_feature)))
+											(compute_on_contained_entities (append filtering_queries (query_quantile feature 0.75 weight_feature)))
 									))
 
 									(assign (assoc
 										mean_absdev
-											(compute_on_contained_entities (list
+											(compute_on_contained_entities (append filtering_queries
```

Comment:
filtering queries should be on the next line, lined up with other queries - same with all the statements below as well. Generally, either they are all queries on one line, or each on a separate line.